app.py

```diff
@@ -189,7 +192,7 @@ def update_graph_content(selected_model):
             dmc.Center(dmc.Text("PPD = 0.06")),
             dmc.Center(dmc.Text("Category = |")),
         ]
-    if selected_model == 'PMV - ASHRAE 55':
```

Comment:
this if should not compare the selected_model to a string but to the model class